backend/src/ai/_ollama.py

```
@@ -0,0 +1,79 @@
+
+
+router = APIRouter(prefix="/ollama", tags=["Ollama"])
+
+class ChatRequest(BaseModel):
+    user_id: str
+    chat_id: str
+    message: str
+    model: str = "deepseek-r1"
+
+
+prompt=('You are a coding assistant running on the Wyra Mood Coder. If you want to run a command on the host machine, '
```

Pylint (3.3.7)

[convention] Line too long (119/100)

(C0301)